uniffi_macros/src/export/trait_interface.rs

```diff
@@ -204,3 +231,31 @@ pub(crate) fn ffi_converter(
         }
     }
 }
+
+pub fn alter_trait(item: &ItemTrait) -> TokenStream {
```

Comment:
It might be nice to have a terse comment explaining what it is doing and specifically what "trait" means in this context. It could also be `pub(crate)` right?